### ISSUE TITLE
Passing token as argument to `mix excoveralls.post` is not working

### ISSUE THREAD

**Issue by knewter** [parroty/excoveralls#38]
If you try, you get an error:

```
$ mix coveralls.post TOKENHERE
Test patterns did not match any file: TOKENHERE
```

**Comment by parroty** [comment]
Thank you for the report. I could confirm that specifying token as mix task argument is not working well, though specifying environment variable (`COVERALLS_REPO_TOKEN`) is working.

I just pushed the fix to master, but I'll confirm the surrounding area later (and then push to hex). 
The format `mix coveralls.post TOKENHERE` conflicts with specifying test file, so I'm making it as 
`--token`.

An example (including other optional parameters) is the following.
```shell
MIX_ENV=test mix coveralls.post --token [YOUR_TOKEN] --branch "master" --name "local host" --commiter "committer name" --sha "fd80a4c" --message "commit message"
```

**Comment by parroty** [comment]
I've pushed the fix as v0.4.5. I'm closing at the moment, but if the problem still remains, please reopen.
